frontend/src/pages/clusterSettings/ClusterSettings.tsx

```diff
@@ -138,15 +139,17 @@ const ClusterSettings: React.FC = () => {
         Number(newClusterSettings?.pvcSize) !== 0 &&
         Number(newClusterSettings?.cullerTimeout) >= MIN_CULLER_TIMEOUT
       ) {
+        setSaving(true);
         updateClusterSettings(newClusterSettings)
           .then((response) => {
+            setSaving(false);
             if (response.success) {
               setClusterSettings(newClusterSettings);
               dispatch(
                 addNotification({
```

Comment:
Oh I omit one thing. Can you also update this to `notification.success` because you update the method? Just a small nit to make a consistent usage in our codebase.

Reply:
I thought of this... Here is my theory on what we are doing to these poorly structured old pages:

I think we need to rework a lot of these pages to support added features that we need. I'm going to leave this as it works fine -- nothing was changed about this notification but the values passed to it -- so I think it's okay to leave as is. This effort will also be impacted by our intents to get rid of Redux.

I think it's best not to get bogged down in reworking aspects when you're not really impacting them. If I had to add a feature to `addNotification` for this to work -- then I'd agree with you. This is a text update and we have other tech debt planned for this area.